packages/web-api/src/response/ViewsOpenResponse.ts

```diff
@@ -2,12 +2,141 @@
 import { WebAPICallResult } from '../WebClient';
 export type ViewsOpenResponse = WebAPICallResult & {
   ok?:                boolean;
+  warning?:           string;
   error?:             string;
-  response_metadata?: ResponseMetadata;
   needed?:            string;
   provided?:          string;
+  view?:              View;
+  response_metadata?: ResponseMetadata;
 };
 
 export interface ResponseMetadata {
   messages?: string[];
 }
+
+export interface View {
+  id?:                    string;
+  team_id?:               string;
+  type?:                  string;
+  title?:                 Close;
+  submit?:                Close;
+  close?:                 Close;
+  blocks?:                Block[];
+  private_metadata?:      string;
+  callback_id?:           string;
+  external_id?:           string;
+  state?:                 State;
+  hash?:                  string;
+  clear_on_close?:        boolean;
+  notify_on_close?:       boolean;
+  submit_disabled?:       boolean;
+  root_view_id?:          string;
+  previous_view_id?:      string;
+  app_id?:                string;
+  app_installed_team_id?: string;
+  bot_id?:                string;
+}
+
+export interface Block {
+  type?:            string;
+  block_id?:        string;
+  label?:           Close;
+  element?:         Element;
+  dispatch_action?: boolean;
+  hint?:            Close;
+  optional?:        boolean;
+  elements?:        Element[];
+  fallback?:        string;
+  image_url?:       string;
+  image_width?:     number;
+  image_height?:    number;
+  image_bytes?:     number;
+  alt_text?:        string;
+  title?:           Close;
+  text?:            Close;
+  fields?:          Close[];
+  accessory?:       Accessory;
+}
+
+export interface Accessory {
+  type?:         string;
+  image_url?:    string;
+  alt_text?:     string;
+  fallback?:     string;
+  image_width?:  number;
+  image_height?: number;
+  image_bytes?:  number;
+}
+
+export interface Element {
+  type?:                            string;
```

Comment:
`type` property values are always defined as far as I'm aware

Reply:
@rr-codes Thanks for the comment. You're right. This `type` and some others are not optional but as we generate the types using code generator (quicktype), this is not customizable.